frontend/simple/views/Mailbox.vue

```diff
@@ -264,3 +259,22 @@ export default {
   }
 }
 </script>
+
+<style lang="sass">
+.signup .level-item
+  margin-top: 10px
+.signup .level.top-align
+  align-items: flex-start
+</style>
```

Comment:
I think .vue files are spec'd to only have one style section (I've never seen them have two). Just to be safe, please move this into the section below (and feel free to use your new CSS classes if you want to get rid of any unnecessary bits if such bits exist).